community/quassel/APKBUILD

```diff
@@ -17,7 +17,7 @@ depends_client="$pkgname-common"
 depends_mono="$pkgname-common"
 depends_server="$pkgname-common qt5-qtbase-postgresql qt5-qtbase-sqlite"
 depends_server_openrc=""
```

Comment:
empty varialbes can be removed for brevity

Reply:
brevity is not a ranking concern